{{ cookiecutter.package_name|slugify }}/README.md

```diff
@@ -18,7 +18,17 @@ To add and install this package as a dependency of your project, run `poetry add
 <summary>Setup: once per device</summary>
 
 {% if cookiecutter.continuous_integration == "GitLab" -%}
-1. [Generate an SSH key](https://docs.gitlab.com/ee/ssh/README.html#generate-an-ssh-key-pair) for GitLab, [add the SSH key to your GitLab account](https://docs.gitlab.com/ee/ssh/README.html#add-an-ssh-key-to-your-gitlab-account), and [add the SSH key to your authentication agent](https://docs.gitlab.com/ee/ssh/README.html#configure-ssh-to-point-to-a-different-directory).
+1. [Generate an SSH key](https://docs.gitlab.com/ee/ssh/README.html#generate-an-ssh-key-pair) and [add the SSH key to your GitLab account](https://docs.gitlab.com/ee/ssh/README.html#add-an-ssh-key-to-your-gitlab-account).
+1. Edit `~/.ssh/config` to configure SSH to automatically load your SSH keys:
```

Comment:
Is there a need to reboot the mac, the shell or the ssh agent for the changes to take effect? I've read Raphael mentioned a reboot step.
If not, that's  good to merge for me :).
Otherwise, it might be useful to have a `2. Reboot your <ssh-agent|mac>` for the config changes to take effect.

Reply:
I think this is not necessary so no change needed here.

Reply:
No reboot required, no (based on my own experience).